sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 - Added experimental scatter gather node to DSL package. This node has a unique mldesigner dependency.
 - Added support to make JobService and ServiceInstance objects serializable when printed
+- Support Singularity compute in pipeline job
```

Comment:
1P only features should be in CHANGELOG?

Reply:
I will keep in mind to update the internal changelog for the 1p change as bugfix.